tests/tools/test_msc.lua

```diff
@@ -308,6 +308,23 @@
 		test.contains("/external:I/usr/local/include", msc.getincludedirs(cfg, cfg.includedirs, cfg.externalincludedirs))
 	end
 
+--
+-- Check handling includedirsafter.
+--
+
+function suite.cflags_onIncludeDirsAfter()
+	includedirsafter { "/usr/local/include" }
```

Comment:
Inhabituel path for msc ^_^
(but ok).

Reply:
Not gonna lie, just stole these from above it in the file